[server] Allow broker to start with default backlogquota in bytes (#11671)
apache/pulsar: site2/docs/reference-configuration.md

|skipBrokerShutdownOnOOM| Flag to skip broker shutdown when broker handles Out of memory error. |false|
|backlogQuotaCheckEnabled|  Enable backlog quota check. Enforces action on topic when the quota is reached  |true|
|backlogQuotaCheckIntervalInSeconds|  How often to check for topics that have reached the quota |60|

> |backlogQuotaDefaultLimitByte| The default per-topic backlog quota limit. Being less than 0 means no limitation. By default, it is -1. | -1 |

Reviewer: ```suggestion
|backlogQuotaDefaultLimitByte| The default per-topic backlog quota limit. A value less than 0 means no limit. | -1 |
```
